RELEASING.md

```diff
@@ -69,10 +69,26 @@ For a discussion on whether to add a prefixing `v` to release tags, see [#35](ht
 
 Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v2.0.0`, as well as `patch` and `backport`. Use release labels to target an issue or a PR for a given release. See [MAINTAINERS](MAINTAINERS.md#triage-open-issues) for more information on triaging issues.
 
-## Releasing
+## Release Scheduling
 
-The OpenSearch release process is centralized.
+The OpenSearch release process is centralized and has several steps along the way.  As part of this process, the Engineering Efficiency team will cut a release ticket and assign an overall release manager for the release.  They will also cut individual tickets per repo for the release with what's expected to go in that release.   
 
+### Release Schedule Date Definitions:
+
+#### Feature freeze
+All code changes that add new capabilities to OpenSearch need to be merged to main and backported to the planned release. Additional merges/backports can happen for bugs fixes and CVEs.  Until we have all plugins using CI, we may continue to have an earlier feature freeze date for OpenSearch core and OpenSearch Dashboards.
```

Comment:
Quote `main`, oxford comma

bugs fixes -> bug fixes

I don't understand the CI sentence, who isn't using CI?!

core -> min

Reply:
By CI I mean continuous integration with the latest version -- i.e. integrating with the "main" of core.  We're still getting requests for core to "freeze" early so plugins can wait to do their integrations until everything is "done".   Is there a better way to express that?

Reply:
I would just remove any special mention of those because this doc applies to all projects in opensearch-project. So delete _Until we have all plugins using CI, we may continue to have an earlier feature freeze date for OpenSearch core and OpenSearch Dashboards._.